### PR TITLE
chore: clean up styling so it renders nicely in iframe

### PR DESCRIPTION
**Pull request by 26damb**
## The basics

- [x] I [validated my changes](https://developers.google.com/blockly/guides/contribute/samples#making_and_verifying_a_change)

## The details
Just cleaning up styling a bit since it appears weird in iframe on devsite

### Resolves

Fixes

### CODE REVIEW

File: examples/devsite-landing-demo/style.css [hunk 1]
```diff
@@ -12,8 +13,10 @@ body {
   --font-family: 'Google Sans';
   --output-max-height: 100%;
   --output-max-width: 340px;
+  --output-min-width: 392px;
```
Comment (cpcallen) [inline examples/devsite-landing-demo/style.css]: It seems surprising that `min-width` > `max-width`, but I'm not CSS expert.

**Review by cpcallen — COMMENTED** [review]
I'm reluctant to approve any changes to devsite-landing-demo at the moment because I'm not sure how to test that they don't break our actual devsite landing page.

I'm going to assign this to @NeilFraser who created this in the first place and presumably knows how to verify the changes (and is also much more famillliar with CSS than I am).

File: examples/devsite-landing-demo/style.css (hunk 1 above)
Comment (26damb) [reply examples/devsite-landing-demo/style.css]: Oops good catch, let me fix that

**Comment by 26damb** [comment]
@cpcallen I was actually the one that created this specific sample, and it will only be utilized on one page. I'm also building out that page myself in staging so I'm seeing how it would look with these change I suggested here.

@cpcallen or @NeilFraser could you reassign to @maribethb that's who I worked with before

It seems I can't reassign myself

**Comment by maribethb** [comment]
This is the new demo created by the marketing folks for the new landing site. This won't affect the demo currently on the site - apologies for the naming confusion, it was originally `devsite-demo2` but the name needed to end in just `demo` for lerna to work and I was fixing it on my day off :P

**Review by maribethb — APPROVED** [review]

**Pull request merged**